backend/server.js

```diff
@@ -1,34 +1,29 @@
 const express = require("express");
 const app = express();
 const mongoose = require("mongoose");
-const cors = require('cors');
+const cors = require("cors");
 const dotenv = require("dotenv");
-const path  = require('path')
-
+const path = require("path");
 dotenv.config();
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
-
 app.use(cors());
 
+const EventRouter = require("./routes/EventsRoutes");
 
-
-mongoose.connect(process.env.DBURI);
+mongoose.connect(process.env.DBURI).then();
 
 const db = mongoose.connection;
-db.on('error', (error) => {
-  console.error('MongoDB connection error:', error);
+db.on("error", (error) => {
+  console.error("MongoDB connection error:", error);
 });
 
-db.once('open', () => {
-  console.log('Connected to MongoDB');
+db.once("open", () => {
+  console.log("Connected to MongoDB");
 });
 
-
-const ParticipantRouter = require("./routes/ParticipantRoutes")
-app.use("/participant", ParticipantRouter)
-
+app.use("/events", EventRouter);
```

Comment:
```suggestion
const EventRouter = require("./routes/EventsRoutes");
app.use("/events", EventRouter);
```

keep the eventRouter and middleware next to each other